__tests__/components/editor/PropertyTemplateOutline.test.js

```diff
@@ -139,7 +139,41 @@ describe('<PropertyTemplateOutline />', () => {
       expect(wrapper.find('div PropertyTypeRow').length).toEqual(1)
       expect(wrapper.find('div PropertyComponent').length).toEqual(1)
     })
+  })
+})
 
+describe('AddResourceTemplate function', () => {
+  const noteResourceTemplate = {
+    "id": "resourceTemplate:bf2:Note",
+    "resourceURI": "http://id.loc.gov/ontologies/bibframe/Note",
+    "resourceLabel": "A Resource Template Note",
+    "propertyTemplates": [
+      {
+        "propertyURI": "http://www.w3.org/2000/01/rdf-schema#label",
+        "propertyLabel": "Note",
+        "mandatory": "false",
+        "repeatable": "false",
+        "type": "literal",
+        "resourceTemplates": [],
+        "valueConstraint": {
+          "valueTemplateRefs": [],
+          "useValuesFrom": [],
+          "valueDataType": {},
+          "editable": "true",
+          "repeatable": "false",
+          "defaults": []
+        }
+      }
+    ]
+  }
+  const propertyTemplate = AddResourceTemplate(noteResourceTemplate, ['resourceTemplate:bf2:Note'])
+
+  it('contains the Resource Template title as the first element', () => {
```

Comment:
I don't understand this description - there is no "title".  I also can't tell from the test if you meant "resourceLabel" as it is only `toBeTruthy()`

Reply:
Changed the description to reflect that we're looking for the `resourceLabel` here and not the title. The Enzyme `matchesElement` function returns a boolean so `toBeTruthy()` provides the correct testing in this case.